src/Symfony/Bridge/Twig/Translation/TwigExtractor.php

```diff
@@ -44,6 +45,11 @@ class TwigExtractor implements ExtractorInterface
      */
     private $twig;
 
+    /**
+     * @var string
+     */
+    private $extension = 'twig';
```

Comment:
This was already discussed [here](https://github.com/symfony/symfony/pull/14002#discussion-diff-26958271R51). I'd prefer to have a class constant, but ok, I'll remove...